

export default async (cb) => {


  try {


  } catch (e) {

    console.log("error", e);

  }
}
